src/genie/libs/parser/iosxr/tests/test_show_mfib.py

```diff
@@ -96,5 +96,117 @@ def test_show_mfib_route_summary_full_custom_vrf(self):
         self.assertEqual(parsed_output, self.golden_parsed_output_3)
 
 
+# ==========================================================================
+# Unittest for 'show mfib platform evpn bucket location'
+# ==========================================================================
+class test_show_mfib_platform_evpn_bucket_location(unittest.TestCase):
+    """ Unit test for show mfib platform evpn bucket location <location>. """
+
+    empty_output = {'execute.return_value': ''}
+
+    golden_output_1 = {'execute.return_value': """
+        Sun May 17 07:47:31.422 UTC
+        LC Type: A9K-4X100GE-TR
+        ------------------------------------------------------
+        ESI Interface        Handle     Bucket ID State Stale 
+        ------------------------------------------------------
+        Bundle-Ether1        0x4000660          0       DF     F 
+        Bundle-Ether1        0x4000660          1      NDF     F 
+        Bundle-Ether1        0x4000660          2       DF     F 
+        Bundle-Ether1        0x4000660          3      NDF     F 
+        Bundle-Ether1        0x4000660          4       DF     F 
+        Bundle-Ether1        0x4000660          5      NDF     F 
+        Bundle-Ether1        0x4000660          6       DF     F 
+        Bundle-Ether1        0x4000660          7      NDF     F 
+        Bundle-Ether1        0x4000660          8       DF     F 
+        Bundle-Ether1        0x4000660          9      NDF     F 
+        Bundle-Ether1        0x4000660         10       DF     F 
+        Bundle-Ether1        0x4000660         11      NDF     F 
+        ------------------------------------------------------"""}
+
+    golden_parsed_output_1 = {
+        'bucket_id':
+            {0:
+                {'esi_interface': 'Bundle-Ether1',
+                 'handle': '0x4000660',
+                 'stale': False,
```

Comment:
let's take `stale`'s value as is.
```suggestion
                 'stale': 'F',
```